content/en/docs/concepts/storage/volume-health-monitoring.md

```diff
@@ -0,0 +1,30 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+--
+title: Volume Health Monitoring 
+content_type: concept
+---
+
+<!-- overview -->
+
+{{< glossary_tooltip text="CSI" term_id="csi" >}} volume health monitoring allows CSI Drivers to detect abnormal volume conditions from the underlying storage systems and report them as events on {{< glossary_tooltip text="PVCs" term_id="persistent-volume-claim" >}} or {{< glossary_tooltip text="Pods" term_id="pod" >}}.
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+<!-- body -->
+
+## Volume Health Monitoring
+
+The Volume Health Monitoring feature is part of Kubernetes implementation of Container Storage Interface (CSI). The Volume Health Monitoring feature is impelmeted in two components: External Health Monitor Controller and Kubelet.
+
+The External Health Monitor Controller will be deployed as a [sidecar container](https://kubernetes-csi.github.io/docs/deploying.html) together with the CSI controller driver. It triggers controller RPC to check the health condition of the CSI volumes. When an abnormal volume condition is detected on a CSI volume, an event will be reported on the PVC.
+
+The external controller sidecar container will also watch for node failure events. This can be enabled by setting the `enable-node-watcher` flag to true. This will only have effects on the local volume type with a CSI driver. When a node failure event is detected, an event will be reported on the PVC to indicate that pods using this PVC are on a failed node.
```

Comment:
"local volume type": is this the "kubernetes.io/local" type?

Reply:
Hmm. Readers might infer that this is describing https://kubernetes.io/docs/concepts/storage/volumes/#local, which is usually written as `local` (and isn't provisioned using CSI).

Reply:
The original intention was for this node watcher to be used only by local PV, however, there isn't specific check to make sure this is only used by local PV so it is really up to the user to decide whether to enable it or not.  I'm going to remove this line about local PV as it can be used by any PV when the flag is enabled.